WalletWasabi/JsonConverters/MaxCoordinationFeeRateJsonConverterNg.cs

```diff
@@ -0,0 +1,33 @@
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using WalletWasabi.Helpers;
+
+namespace WalletWasabi.JsonConverters;
+
+public class MaxCoordinationFeeRateJsonConverterNg : JsonConverter<Fraction>
+{
+	/// <inheritdoc />
+	public override Fraction Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
+	{
+		// For backward compatibility, MaxCoordinationFeeRate used to be a percentage
+		// If it was migrated already then TokenType will be a string
+		if (reader.TokenType == JsonTokenType.Number)
+		{
+			return new Fraction(reader.GetDecimal(), 100);
+		}
+
+		var stringValue = reader.GetString();
+		var splitValue = stringValue!.Split('/');
```

Comment:
I think the config file should not contain 1/3 but 0.3333334 even when the fractional representation is better, this is because we are using the decimal representation everywhere.